frontend/STRUCTURE.md

```
@@ -0,0 +1,52 @@
+# Frontend Code Structure
+
+This document describes the key folders and files inside the `frontend/` directory.
+
+## Quick start
+- Install: `npm i`
+- Run locally: `npm run dev` (opens at http://localhost:3000)
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)